app/layout.tsx

```diff
@@ -64,8 +63,7 @@ export default function RootLayout({ children }: LayoutProps) {
         <body>
           <FlexBox justifyContent="between" direction="col" className="min-h-screen relative">
             <Skiplink />
-            <Masthead />
-            <main id="main-content">{children}</main>
```

Comment:
The `Masthead` has been absorbed into `children` which is the top level Storyblok page components (our page components - basic page, story, homepage). We have different version of the `Masthead` depending on what Story Hero is picked so having it inside the page components makes it easier to render conditional styles for the `Masthead`.